package.json

```diff
@@ -16,8 +16,9 @@
     "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' nyc mocha --require ts-node/register --ui bdd 'test/**/*.spec.ts'",
     "test:cov": "nyc --reporter=lcov npm run test --reporter mocha-sonarqube-reporter --exit",
     "sonar": "sonar-scanner",
-    "cucumber": "cucumber-js --require-module ts-node/register --require features/support/test-setup.ts --require features/support/steps/**/*.ts --publish-quiet --tags 'not @wip'",
+    "cucumber": "cucumber-js --require-module ts-node/register --require features/support/test-setup.ts --require features/support/steps/**/*.ts --tags 'not @wip'",
```

Comment:
Why is --publish-quiet removed ?

Reply:
`publishQuiet` option is no longer needed, you can remove it from your configuration; see https://github.com/cucumber/cucumber-js/blob/main/docs/deprecations.md

Keep seeing this warning in the console